spec/features/user_creates_new_session_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+feature "user creates new session" do
+  scenario "successfully" do
+    user = create(:user)
+    visit login_path
+
+    fill_in "Email", with: user.email
```

Comment:
Remove extra line.

The idea is to separate setup, from action. Visit is an action in my mind

Reply:
Also use the field's id or name rather than the label.
(or use a translation for the label)

Reply:
Done in sha: e47bdda